src/StatusQ/Components/StatusWizardStepper.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.12
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Item {
+    id: wizardWrapper
+    width: parent.width
+    height: 56
+    property ListModel stepsModel: ListModel { }
+
+    ListView {
+        id: repeat
+        width: childrenRect.width
+        height: parent.height
+        anchors.horizontalCenter: parent.horizontalCenter
+        orientation: ListView.Horizontal
+        model: stepsModel
+        delegate: Item {
+            id: wrapperItem
+            width: (index === 0) ? descriptionLabel.contentWidth : (wizardWrapper.width / repeat.count)
+            height: 56
+            onXChanged: {
+                //as x changes while delegates are created, direct assignment doesn't work
+                x = (index === (repeat.count-1)) ? (width+32) : (label.width/2) + 8
+            }
+            Rectangle {
```

Comment:
Just a suggestion... We have the `StatusProgressBar` control which inherits from `ProgressBar` which could maybe be used here (instead of using Rectangles) and that component could take advantage of its behavior. Please take a look at it and let me know your thoughts!

Reply:
Fully agree!

Reply:
ok, nevertheless I had to add another property as the component was not 100% fitting.

Reply:
Are there any other things that would make using `StatusProgressBar` hard to use here?
Cause then it might be worth to explore teaching `StatusProgressBar` a custom border color, so the component can be reused here.

At least visually, it really does look like this is essentially a progress bar.

Reply:
I think progress bar is ok just by adding a new `backgroundBorderColor` property (which I already did).